components/homme/test_execs/preqx-nlev72-kokkos/CMakeLists.txt

```diff
@@ -14,11 +11,17 @@ createTestExec(preqx-nlev72-kokkos preqx_kokkos 4 4 72 FALSE FALSE 35)
 SET (HOMME_TEST_VCOORD_INT_FILE acme-72i.ascii)
 SET (HOMME_TEST_VCOORD_MID_FILE acme-72m.ascii)
 SET (HOMME_TEST_NE 2)
+SET (HOMME_TEST_TIME_STEP 300)
 SET (HOMME_TEST_NDAYS 1)
 SET (HOMME_TEST_QSIZE 4)
 SET (HOMME_TEST_RSPLIT 3)
 SET (HOMME_TEST_MOISTURE notdry)
 SET (HOMME_TEST_TIME_STEP 600)
+SET (HOMME_TEST_HVSCALING 0)
```

Comment:
why do you need these changed for preqx xx ?

Reply:
I think I added it cause the `preqx.nl` template namelist used to generate the tests has that setting too, so I didn't want to have entries in the namelist that look like `blah =` (without a rhs). Is that param not used by preqx?